binance_trade_bot/backtest.py

```diff
@@ -39,6 +73,33 @@ def increment(self, interval=1):
     def get_fee(self, origin_coin: Coin, target_coin: Coin, selling: bool):
         return 0.0075
 
+    def get_historical_klines(self, ticker_symbol='ETCUSDT', interval='1m', target_date=None, end_date=None, limit=None,
+                              frame='daily'):
+        fromdate = datetime.strptime(target_date, "%d %b %Y %H:%M:%S")  # - timedelta(days=1)
+        r = requests.get(
+            f'https://s3-ap-northeast-1.amazonaws.com/data.binance.vision?delimiter=/&prefix=data/spot/{frame}/klines/{ticker_symbol}/{interval}/',
```

Comment:
Why not using [params argument](https://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls)?